### PR TITLE
Zoom slider

### PR DESCRIPTION
**Pull request by nicoring**
adds a slider as the first part of the new zooming experience. Changing the slider calls the `zoomTo: aNumber` method on `SClassDiagram` with signals. Refactors the zooming step from telling the scrollpane the change values to setting absolute zoomLevels on the scrollpane.

### CODE REVIEW

File: packages/Path-View-Core.package/SClassDiagram.class/class/defaultZoomLevel.st [hunk 1]
```diff
@@ -0,0 +1,4 @@
+constants
+defaultZoomLevel
+
```
Comment (01Track1mp3) [inline packages/Path-View-Core.package/SClassDiagram.class/class/defaultZoomLevel.st]: can you put a method comment which explains the semantics of zoomLevel?
is it actual scale?

File: packages/Path-View-Core.package/SScrollPane.class/instance/getCenter.st [hunk 2]
```diff
@@ -0,0 +1,4 @@
+zooming
+getCenter
+
+	^ (self scroller offset + (self scroller extent / 2)) / self scroller scale
```
Comment (01Track1mp3) [inline packages/Path-View-Core.package/SScrollPane.class/instance/getCenter.st]: get center of what?

File: packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testZoomingWithSlider.st [hunk 3]
```diff
@@ -0,0 +1,12 @@
+zooming
+testZoomingWithSlider
+
+	| pathView slider diagram newZoomLvl |
+
+	pathView := SPathView privateNew buildWith: MorphicToolBuilder new.
+	slider := (pathView allSubmorphsOfKind: UiSlider) asOrderedCollection first.
+	diagram := (pathView allSubmorphsOfKind: SClassDiagram) asOrderedCollection first.
+	newZoomLvl := 0.5. "Should not be default value"
```
Comment (01Track1mp3) [inline packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testZoomingWithSlider.st]: what does this mean?

**Comment by 01Track1mp3** [comment]
just these small adaptions. lovely code otherwise!

File: packages/Path-View-Core.package/SScrollPane.class/instance/getCenter.st (hunk 2 above)
Comment (nicoring) [inline packages/Path-View-Core.package/SScrollPane.class/instance/getCenter.st]: `getUnScaledCenter`

File: packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testZoomingWithSlider.st (hunk 3 above)
Comment (Chaoste) [inline packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testZoomingWithSlider.st]: The slider is set to "0.5". The test is successfully if the parent morph (ScrollPane) contains the same value

**Pull request merged**